geometry_msgs/PoseStamped image_goal
---
bool success
geometry_msgs/PoseStamped map_goal
